src/pages/Introduce/Introduce.scss

```
@@ -0,0 +1,61 @@
+@import "../../style/mixin.scss";
+@import "../../style/variables";
+
+* {
+  box-sizing: border-box;
+}
+
+.introducename {
+  @include flexCenter;
+
+  .container {
+    border: 1px solid #ddd;
+    border-radius: 12px;
+    padding: 30px;
+  }
+
+  .container .logobox {
+    width: 240px;
+    height: 240px;
+    margin-bottom: 30px;
+    background-color: black;
+    background-size: cover;
+    background-repeat: no-repeat;
+    border-radius: 12px;
+    text-align: center;
+  }
```

```suggestion
  .container {
    border: 1px solid #ddd;
    border-radius: 12px;
    padding: 30px;
    
    .logobox {
    width: 240px;
    height: 240px;
    margin-bottom: 30px;
    background-color: black;
    background-size: cover;
    background-repeat: no-repeat;
    border-radius: 12px;
    text-align: center;
  }
  }
```
scss의 nesting을 활용해서 이런 방식으로 nesting 할 수 있겠습니다!

---

넵!